app/assets/stylesheets/config/_colors.css

```diff
@@ -276,13 +276,27 @@
   --indicator-info-bg: var(--accent-brand);
   --indicator-info-color: rgb(var(--white));
 
+  /***********************************************
+  ** List items **********************************
+  ***********************************************/
+  --list-item-bg-hover: rgb(var(--grey-50));
+
+  /***********************************************
+  ** Colors **************************************
+  ***********************************************/
+  --color-primary: rgb(var(--grey-900));
+  --color-secondary: rgb(var(--grey-500));
```

Comment:
No worries, I have it under control...

![](https://media.giphy.com/media/Z9P7Kbj8pjVxuFh9UT/giphy.gif)